src/Chronos.php

```diff
@@ -182,6 +196,112 @@ public static function hasTestNow(): bool
         return static::$testNow !== null;
     }
 
+    /**
+     * Sets the time.
+     *
+     * @param int $hours Hours of the time
+     * @param int $minutes Minutes of the time
+     * @param int $seconds Seconds of the time
+     * @param int $microseconds Microseconds of the time
+     * @return static
+     */
+    public function setTime(int $hours, int $minutes, int $seconds = 0, int $microseconds = 0): static
```

Comment:
Can we have all arguments default to 0? That would allow one to do for e.g. `->setTime(minutes: 1)`.

Reply:
We can. This matched DateTimeImmutable::setTime(). I'm not sure why it was defaulting to null and not 0.